.ci/.jenkins-schema.yml

```diff
@@ -0,0 +1,17 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+    SPEC_FILEPATH: "test/Elastic.Apm.Tests/TestResources/json-spec"
+  - REPO: "apm-agent-go"
+    SPEC_FILEPATH: "internal/apmschema/jsonschema"
+  - REPO: "apm-agent-java"
+    SPEC_FILEPATH: "apm-agent-core/src/test/resources/apm-server-schema/current"
+  - REPO: "apm-agent-nodejs"
+    SPEC_FILEPATH: "test/integration/api-schema/apm-server-schema"
+  - REPO: "apm-agent-php"
+    SPEC_FILEPATH: "tests/APM_Server_intake_API_schema/latest_used"
+  - REPO: "apm-agent-python"
+    SPEC_FILEPATH: "tests/upstream/json-specs"
+  - REPO: "apm-agent-ruby"
+    SPEC_FILEPATH: "spec/fixtures"
+  - REPO: "apm-agent-rum-js"
```

Comment:
@jahtalab and @vigneshshanmugam , I need your input here, where should the https://github.com/elastic/apm-server/tree/master/docs/spec/v2/ be copied to?

Reply:
@vigneshshanmugam , just confirmed we don't need these files since they use a docker apm-server to test with
```suggestion
```

Reply:
Just to confirm what we spoke offline, RUM agent does not fetch schema files for testing. We test against a local version of the APM server. We don't need to copy any files for the RUM agent atleast for now.